src/pages/teachers/key-stages/[keyStageSlug]/subjects/index.tsx

```diff
@@ -168,7 +168,8 @@ export const getStaticProps: GetStaticProps<
 
         const combinedMaths: KeyStageSubjectData = {
           programmeSlug: newMaths.programmeSlug,
-          programmeCount: newMaths.programmeCount + legacyMaths.programmeCount,
+          programmeCount:
+            newMaths.programmeCount + legacyMaths.programmeCount - 1,
```

Comment:
thinking about this I think it should actually be that we take the higher programme count? otherwise if each have 2 it will become 3, if each had 6 it would become 11 but there are still only 6 programmes? Might get weird if they have different programes...but I don't think we need to worry about that

Reply:
Changed